packages/core/src/services/files/upload.ts

```diff
@@ -38,10 +66,9 @@ export async function uploadFile(
 
   try {
     await disk.putFile(key, file).then((r) => r.unwrap())
-    const url = await disk.getSignedUrl(key, {
-      expiresIn: undefined,
-      contentDisposition: 'inline',
-    })
+    // TODO: Use temporal signed URLs, with a (micro)service
+    // acting as a reverse proxy refreshing the signed urls
```

Comment:
Don't put this TODO, I don't think we'll do in the short term no?

Reply:
Just for future reference so we don't forget!

Reply:
Don't put too much todos or team morale will lower 😂 Just the necessary